packages/block-editor/src/components/media-placeholder/index.js

```diff
@@ -110,24 +110,54 @@ export class MediaPlaceholder extends Component {
 
 	onFilesUpload( files ) {
 		const {
-			addToGallery,
 			allowedTypes,
 			mediaUpload,
 			multiple,
 			onError,
 			onSelect,
-			value = [],
```

Comment:
Previously value was not required and the default was empty. Now we are using a default value and we have the following code `const currentMedia = this.props.value;` and then `const filteredMedia = currentMedia.filter( ( item ) => {` this change will make all the existing plugins using MediaPlaceholder without passing a value (were relying on the previous default value) crash because in that case this.props.value is undefined and then we call undefined(currentMedia).filter.

Reply:
Good call.  I re-added the fallback for the empty array.  I had removed it due to changes I made in how the Gallery component passes down the value, but it makes sense to keep it regardless of that.